deploy/aws-lambda/variables.tf

```
@@ -0,0 +1,9 @@
+variable "secrethub_mutating_webhook_version" {
+  type = string
+  default = "v0.2.0"
```

Could use a description:
```suggestion
  description = "Version of the SecretHub webhook to deploy."
  default     = "v0.2.0"
```

---

Solved